docs/csharp/whats-new/tutorials/static-abstract-interface-methods.md

```diff
@@ -29,17 +29,17 @@ public static double MidPoint(double left, double right) =>
     (left + right) / (2.0);
 ```
 
-The same logic would work for any numeric type: `int`, `short`, `long`, `float` `decimal`, or any type that represents a number. You need to have a way to use the `+` operator, `/` operator, and define a value for `2`. You can use the `System.INumber` interface in the `System.Runtime.Experimental` NuGet package to write the preceding method as the following generic method:
+The same logic would work for any numeric type: `int`, `short`, `long`, `float` `decimal`, or any type that represents a number. You need to have a way to use the `+` and `/` operators, and to define a value for `2`. You can use the <xref:System.Numerics.INumber%601?displayProperty=fullName> interface to write the preceding method as the following generic method:
 
 :::code language="csharp" source="./snippets/staticinterfaces/Utilities.cs" id="MidPoint":::
 
-Any type that implements the `INumber` interface must include a definition for `operator +`, and `operator /`. The denominator is defined by `(T.One + T.One)` to create the value `2` for any numeric type. The readonly properties `Zero` and `One` are also implemented by every type that implements `INumber`. Using that property rather than `2` forces the denominator to be the same type as the two parameters. This implementation has the potential for overflow if `left` and `right` are both large enough values. There are alternative algorithms that can avoid this potential issue.
+Any type that implements the <xref:System.Numerics.INumber%601> interface must include a definition for `operator +`, and for `operator /`. The denominator is defined by `(T.One + T.One)` to create the value `2` for any numeric type. The readonly properties `Zero` and `One` are also implemented by every type that implements `INumber`. Using that property rather than `2` forces the denominator to be the same type as the two parameters. (This implementation has the potential for overflow if `left` and `right` are both large enough values. There are alternative algorithms that can avoid this potential issue.)
```

Comment:
`T.CreateChecked(2)`, `T.CreateSaturating(2)`, or `T.CreateTruncating(2)` are also ways you can achieve the `(T.One + T.One)` logic.

For more complex values, it would likely be the preferred mechanism.

Reply:
`CreateChecked` - This one will throw an overflow exception if the input cannot fit in the destination
`CreateSaturating` - This one will "clamp" to `T.MinValue`/`T.MaxValue` if the input cannot fit in the destination
`CreateTruncating` - This one will "wrap" around if the input cannot fit in the destination.

For an example, given a too large value:
* `byte.CreateChecked(384)` will throw `OverflowException`
* `byte.CreateSaturating(384)` will result in `255` because it is greater than `byte.MaxValue` (which is `255`)
* `byte.CreateTruncating(384)` will result in `128` because it takes the lowest 8-bits (`384` has a hex representation of `0x0180` and the lowest 8-bits is `0x80` which is `128`)

For example, Given a too small value:
* `byte.CreateChecked(-384)` will throw `OverflowException`
* `byte.CreateSaturating(-384)` will result in `0` because it is greater than `byte.MinValue` (which is `0`)
* `byte.CreateTruncating(-384)` will result in `128` because it takes the lowest 8-bits (`384` has a hex representation of `0xFE80` and the lowest 8-bits is `0x80` which is `128`)

There are some special considerations for IEEE 754 floating-point types, like `float` and `double`, as they have special values `PositiveInfinity`, `NegativeInfinity`, and `NaN`. All three APIs behave as `Saturate` instead. Likewise, while `MinValue` and `MaxValue` represent the largest negative/positive "normal" number, the actual min/max values are `NegativeInfinity` and `PositiveInfinity`, so they clamp to these values instead.

Reply:
Thanks, I'll add this info. BTW, do you know why the Create* methods never come up in IntelliSense (code completion)?